tribe-ext-eventbrite-addl-opts/index.php

```diff
@@ -0,0 +1,434 @@
+			|| ! array_key_exists( $locale, $this->eb_locale_country_name_array() )
+		) {
+			$locale = 'en_US';
+		}
+
+		return $locale;
+	}
+
+	/**
+	 * Get Eventbrite currency from specified locale
+	 * This list is hand-crafted from noting the currency specified in the calculator at https://www.eventbrite.com/fees/ (per TLD) after selecting each region from Eventbrite's Region Selector. Could just play with PHP's https://secure.php.net/localeconv but decided against it to avoid changing system settings.
+	 * @access private
+	 * @return string
+	 */
+	private function get_currency_from_eb_locale() {
+		$locale = $this->get_chosen_eb_locale();
+
+		$locale_currencies = array(
+			'es_AR'		=> 'ARS',
+			'en_AU'		=> 'AUD',
+			'pt_BR'		=> 'BRL',
+			'en_CA'		=> 'CAD',
+			'fr_CA'		=> 'CAD',
+			'de_DE'		=> 'EUR',
+			'es_ES'		=> 'EUR',
+			'fr_FR'		=> 'EUR',
+			'en_HK'		=> 'HKD', // EB's fees calculator does not display
+			'en_IE'		=> 'EUR',
+			'it_IT'		=> 'EUR',
+			'nl_NL'		=> 'EUR',
+			'en_NZ'		=> 'NZD',
+			'pt_PT'		=> 'EUR',
+			'en_SG'		=> 'SGD', // EB's fees calculator does not display
+			'en_GB'		=> 'GBP',
+			'en_US'		=> 'USD',
+		);
+
+		$currency = $locale_currencies[$locale];
+
+		return $currency;
+	}
+
+	/**
+	 * Get Eventbrite Top-Level Domain (TLD) from specified locale
+	 * This list is hand-crafted from noting the domain change after selecting each region from Eventbrite's Region Selector.
+	 * @access private
+	 * @return string
+	 */
+	private function eb_locale_to_tld() {
+		$locale = $this->get_chosen_eb_locale();
+
+		$locale_tlds = array(
+			'es_AR'		=> '.com.ar',
+			'en_AU'		=> '.com.au',
+			'pt_BR'		=> '.br',
+			'en_CA'		=> '.ca',
+			'fr_CA'		=> '.ca', // it is the same!
+			'de_DE'		=> '.de',
+			'es_ES'		=> '.es',
+			'fr_FR'		=> '.fr',
+			'en_HK'		=> '.hk',
+			'en_IE'		=> '.ie',
+			'it_IT'		=> '.it',
+			'nl_NL'		=> '.nl',
+			'en_NZ'		=> '.co.nz',
+			'pt_PT'		=> '.pt',
+			'en_SG'		=> '.sg',
+			'en_GB'		=> '.co.uk',
+			'en_US'		=> '.com',
+		);
+
+		$tld = $locale_tlds[$locale];
+
+		return $tld;
+	}
+
+	/**
+	 * Get Eventbrite TLD from user-chosen Eventbrite Locale
+	 * @access private
+	 * @return string
+	 */
+	private function get_eb_tld() {
+		return $this->eb_locale_to_tld( $this->get_chosen_eb_locale() );
+	}
+
+	// originally from https://theeventscalendar.com/knowledgebase/configuring-eventbrite-tickets-to-use-eventbrite-co-uk/
+	public function api_url( $url ) {
+		$replace_with = $this->get_eb_tld();
+
+		$new_url = str_replace( '.com', $replace_with, $url );
+
+		return esc_url( $new_url );
+	}
+
+	// originally from https://tribe.slack.com/archives/products-support/p1485534655007614?thread_ts=1485534292.007600&cid=C2ZAM3ECF
+	// which was for https://theeventscalendar.com/support/forums/topic/why-does-plugin-not-work-for-private-events-yet-iframe-widget-does/
+	/**
+	 * Displays Eventbrite Tickets iframe even if the Eventbrite event is Private
+	 * @return string|void
+	 */
```

Comment:
You could change this comment and the rest to this format. This way you have some context right away about what the method is.  There is a couple like that in this extension and it would be nice to have comments even a short title for every method.

/**
 * Displays Eventbrite Tickets iframe even if the Eventbrite event is Private
 * @return string|void
 * 
 * originally from https://tribe.slack.com/archives/products-support/p1485534655007614?thread_ts=1485534292.007600&cid=C2ZAM3ECF
 * which was for https://theeventscalendar.com/support/forums/topic/why-does-plugin-not-work-for-private-events-yet-iframe-widget-does/
 */